crates/cli/tests/it/main.rs

```diff
@@ -30,6 +30,9 @@ pub(crate) mod forge_utils;
 #[cfg(feature = "external-integration-tests")]
 mod integration;
 
+#[cfg(feature = "maple-integration-test")]
```

Comment:
maybe we want more, so we should rename this

Reply:
done on https://github.com/foundry-rs/foundry/pull/5611/commits/b4c7dc10fe4895d0884a76de65859bf1ae8efaec & https://github.com/foundry-rs/foundry/pull/5611/commits/fb6a87e074cf193483ab2dbd3727706f95045c1e